configs/_config.checks.yml

```diff
@@ -0,0 +1,67 @@
+html_proofer:
```

Comment:
Is there a reason why we can't keep this config at the root of the project like all other config YAMLs?

Reply:
No, we can still keep it there
This was a part of my suggestion to move all the special configs (`.prod.yml`, `.local.yml`, etc) into the `config` directory; and keep only `_config.yml` at the root.
Contributors won't need these configs in most cases, so no need to expose them at the root level.